WalletWasabi.Fluent/Styles/Themes/BaseDark.axaml

```diff
@@ -2,15 +2,19 @@
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
   <Styles.Resources>
     <Color x:Key="SystemAccentColor">#78A827</Color>
+    <Color x:Key="SystemAccentColorLight1">#719E26</Color>
+    <Color x:Key="SystemAccentColorDark1">#608522</Color>
+    <Color x:Key="SystemControlDisabledBaseLowBrush">#80A3A3A3</Color>
+
     <Color x:Key="LineChartAreaGradientStartStopColor">#FF78A827</Color>
     <Color x:Key="LineChartAreaGradientEndStopColor">#0078A827</Color>
     <Color x:Key="LineChartLineColor">#78A827</Color>
 
-    <SolidColorBrush x:Key="ActionButtonForegroundColor">#FFFFFF</SolidColorBrush>
+    <SolidColorBrush x:Key="ActionButtonForegroundColor" Color="White" />
     <SolidColorBrush x:Key="ActionButtonBackgroundColor" Color="{DynamicResource SystemAccentColor}" />
-    <SolidColorBrush x:Key="ActionButtonPressedBackgroundColor" Color="#59B164" />
-    <SolidColorBrush x:Key="ActionButtonPointerOverBackgroundColor" Color="#90CB9C" />
-    <SolidColorBrush x:Key="ActionButtonDisabledBackgroundColor" Color="#80A3A3A3" />
+    <SolidColorBrush x:Key="ActionButtonPressedBackgroundColor" Color="{DynamicResource SystemAccentColorDark1}" />
+    <SolidColorBrush x:Key="ActionButtonPointerOverBackgroundColor" Color="{DynamicResource SystemAccentColorLight1}" />
+    <SolidColorBrush x:Key="ActionButtonDisabledBackgroundColor" Color="{DynamicResource SystemControlDisabledBaseLowBrush}" />
```

Comment:
These are all the same in light and dark themes. Can you move them into the `Base.axaml`?